.github/actions/step_generate_configs/action.yml

```diff
@@ -11,7 +11,6 @@ runs:
     working-directory: ${{ github.workspace }}
     shell: pwsh
     run: |
-      conda activate release-env
```

Comment:
Did we measure how much faster after this change?

Reply:
![image](https://github.com/microsoft/promptflow/assets/2208599/95232c76-4018-4fd0-bce3-ac935cd52f76)
25s in total,

And I take one old pipeline
![image](https://github.com/microsoft/promptflow/assets/2208599/3b2bd31f-2729-4db8-bad7-018a44504d93)

This is 3min 30s